src/helper/task.ts

```diff
@@ -265,10 +265,27 @@ export class ZkWasmServiceHelper {
       console.log("json", queryJson);
     }
 
-    let tasks = await this.endpoint.invokeRequest("GET", `/tasks`, queryJson);
+    let tasks = (await this.endpoint.invokeRequest(
+      "GET",
+      `/tasks`,
+      queryJson
+    )) as PaginationResult<Task[]>;
     if (this.endpoint.enable_logs) {
       console.log("loading task board!");
     }
+    // Convert the corresponding proof fields into Uint8Array from number[] (json)
+
+    tasks.data.forEach((task) => {
+      task.instances = new Uint8Array(task.instances);
+      task.proof = new Uint8Array(task.proof);
+      task.batch_instances = new Uint8Array(task.batch_instances);
+      task.shadow_instances = new Uint8Array(task.shadow_instances);
+      task.single_proof = new Uint8Array(task.single_proof);
+      task.aux = new Uint8Array(task.aux);
+      // May as well also convert the external host table to Uint8Array
+      task.external_host_table = new Uint8Array(task.external_host_table);
```

Comment:
in task we have other field with defined as Uint8Array, all of them need internal converting.
Like:
input_context
output_context
task_fee